docs/Project and Community/FAQ.rst

```diff
@@ -53,23 +51,23 @@ requires.
 Installation
 ------------
 
-ZFS on Linux is available for all major Linux distributions. Refer to
-the :doc:`getting started <../Getting Started/index>` section of the wiki for links to installations
-instructions for many popular distributions. If your distribution isn't
-listed you can always build ZFS on Linux from the latest official
-`tarball <https://github.com/zfsonlinux/zfs/releases>`__.
+OpenZFS is available for all major Linux distributions and FreeBSD. Refer to
+the :doc:`getting started <../Getting Started/index>` section of the wiki for
+links to installations instructions. If your distribution/OS isn't
+listed you can always build OpenZFS from the latest official
+`tarball <https://github.com/openzfs/zfs/releases>`__.
 
 Supported Architectures
 -----------------------
 
-ZFS on Linux is regularly compiled for the following architectures:
+OpenZFS is regularly compiled for the following architectures:
 x86_64, x86, aarch64, arm, ppc64, ppc.
 
-Supported Kernels
+Supported Linux Kernels
 -----------------
```

Comment:
other than these "Title underline too short." warnings this looks good

Reply:
Ouch, fixed